### ISSUE TITLE
Failed Proxmox VM build: './scripts/shaderinclude.py': No such file or directory

### ISSUE THREAD

**Issue by scaredmushroom** [nix-community/nixos-generators#235]
I'm trying to build a Proxmox image from nixpkgs master (latest commit: 6f6f750d8065473c7d0eeefad414c77e75797513).

Command used:
`nix run github:nix-community/nixos-generators -- --format proxmox --configuration configuration.nix -I nixpkgs=/{PATH}/nixpkgs/`

Expected outcome:
VMA container image for Proxmox.

Actual outcome:
Build failure with the last 10 log lines being 
```shell
       last 10 log lines:
       > patching file meson.build
       > patching file vma-reader.c
       > patching file vma-writer.c
       > patching file vma.c
       > patching file vma.h
       > applying patch /nix/store/z52sm8m0skkrsdsynw3kq0pnm6hhvgdb-inline.patch
       > patching file vma-writer.c
       > configuring
       > chmod: cannot access './scripts/shaderinclude.py': No such file or directory
       > /nix/store/37p8gq9zijbw6pj3lpi1ckqiv18j2g62-stdenv-linux/setup: line 136: pop_var_context: head of shell_variables not a function context
       For full logs, run 'nix log /nix/store/95f6pri1v24cjz8x92b5l0hpcx928i5b-qemu-host-cpu-only-7.0.0.drv'.
error: 1 dependencies of derivation '/nix/store/3axjlp880wvzci86g9afbjnghdpn3y29-proxmox-nixos-23.11pre-git.drv' failed to build
```

Full log:
https://pastebin.com/raw/nAPzjdbj

Full log for qemu-host-cpu-only:
```shell
unpacking sources
unpacking source archive /nix/store/gpvz4dnwwjryzcky19zlzcd261pbaw6k-qemu-7.0.0.tar.xz
source root is qemu-7.0.0
setting SOURCE_DATE_EPOCH to timestamp 1650395499 of file qemu-7.0.0/roms/edk2/UnitTestFrameworkPkg/Library/CmockaLib/cmocka/tests/test_wildcard.c
patching sources
applying patch /nix/store/38b9x117z1wh3nbyj2pmlcrznj7xd8ds-0026-PVE-Backup-add-vma-backup-format-code.patch
patching file block/meson.build
Hunk #1 succeeded at 45 with fuzz 1 (offset -1 lines).
patching file meson.build
patching file vma-reader.c
patching file vma-writer.c
patching file vma.c
patching file vma.h
applying patch /nix/store/z52sm8m0skkrsdsynw3kq0pnm6hhvgdb-inline.patch
patching file vma-writer.c
configuring
chmod: cannot access './scripts/shaderinclude.py': No such file or directory
/nix/store/37p8gq9zijbw6pj3lpi1ckqiv18j2g62-stdenv-linux/setup: line 136: pop_var_context: head of shell_variables not a function context
```

Configuration.nix:
```nix
{
  config,
  pkgs,
  lib,
  ...
}: {
  services = {
    cloud-init.network.enable = true;
    sshd.enable = true;
    openssh.permitRootLogin = lib.mkDefault "yes";
    openssh.passwordAuthentication = lib.mkDefault false;
    getty.autologinUser = lib.mkDefault "root";
  };
  users.users.root = {
    openssh.authorizedKeys.keys = [
      "......"
    ];
  };
  system.stateVersion = "22.11";
}
```

**Comment by scaredmushroom** [comment]
This problem is reproducible on nixos-23.05 but not on nixos-22.11.

**Comment by scaredmushroom** [comment]
NixOS/nixpkgs#233517

**Comment by illustris** [comment]
Hey. This should be fixed once that PR is merged. I just need to test Artturin's suggestions and update it.

**Comment by illustris** [comment]
@scaredmushroom this issue can be closed now, right?

**Comment by scaredmushroom** [comment]
Yes. This was fixed with the new 23.05 NixOS release.
